tests/selenium/pbtest.py

```diff
@@ -259,7 +259,10 @@ def run(self, result=None):
                 if i == nretries - 1:
                     raise
                 else:
-                    print('\nRetrying {} ...'.format(result))
+                    wait_secs = 2 ** i
+                    print('\nRetrying {} after {} seconds ...'.format(
+                        result, wait_secs))
+                    time.sleep(wait_secs)
```

Comment:
This wait period happens between the when selenium shuts down, and when it restarts. It should not effect the environment in which the test is run. Is that intended?